src/uct/sm/scopy/cma/cma_iface.c

```diff
@@ -100,8 +100,12 @@ uct_cma_iface_is_reachable_v2(const uct_iface_h tl_iface,
      * permissions are enforced before the remote address sanity checks.
      */
     peer_pid = iface_addr->super.id & ~UCT_CMA_IFACE_ADDR_FLAG_PID_NS;
-    return (process_vm_readv(peer_pid, &iov, 1, &iov, 1, 0) != -1) ||
-           (errno != EPERM);
+    if ((process_vm_readv(peer_pid, &iov, 1, &iov, 1, 0) == -1) &&
+           (errno == EPERM)) {
```

Comment:
indentation

Reply:
will amend the merge commit and force-push it